emsdk.py

```diff
@@ -522,9 +522,10 @@ def fix_potentially_long_windows_pathname(pathname):
 # race-free way to do it. This method removes the destination if it exists, so
 # the move always works
 def move_with_overwrite(src, dest):
+  # debug_print('move_with_overwrite(src=' + src + ', dest=' + dest + ')')
   if os.path.exists(dest):
     os.remove(dest)
-  os.rename(src, dest)
+  shutil.move(src, dest)
```

Comment:
Is this a separate issue?

Reply:
Given i did used `os.getenv('TEMP')` as the target where to extract the files under Windows, `os.rename()` may does fail, thus moving to `shutil.move()` to deal with the case where the target and destination are not in the same drive (if they are, it will invoke `os.rename()` anyway as you sure know)